### PR TITLE
Added Infinity division

### PR DESCRIPTION
**Pull request by CodeMaxx**
Added `Infty::div` with testcases. @isuruf

### CODE REVIEW

File: symengine/infinity.cpp [hunk 1]
```diff
@@ -126,10 +126,20 @@ RCP<const Number> Infty::mul(const Number &other) const
     }
 }
 
-// TODO
 RCP<const Number> Infty::div(const Number &other) const
 {
-    return zero;
+    if(is_a<Infty>(other)) {
+        throw std::runtime_error("Indeterminate Expression: "
```
Comment (isuruf) [inline symengine/infinity.cpp]: Use `UndefinedError` here.

Comment (CodeMaxx) [inline symengine/infinity.cpp]: Oh nice we have SymEngineException handler now 😄 Updated.

**Comment by CodeMaxx** [comment]
@isuruf I can't understand the Travis error. I don't think its related to the code. Will you please check?

**Comment by certik** [comment]
@CodeMaxx it's caused by incorrect formatting in your PR. Just apply the patch from the bottom of https://travis-ci.org/symengine/symengine/jobs/167043905.

**Comment by certik** [comment]
There seem to be other errors as well, those seem unrelated to this PR.

**Comment by CodeMaxx** [comment]
Oh right I forgot to apply clang-format. Fixed.

**Comment by certik** [comment]
The other error seems to be caused by #1091.

**Comment by isuruf** [comment]
@CodeMaxx, can you merge with master?

**Comment by CodeMaxx** [comment]
Done

**Comment by isuruf** [comment]
Thanks

**Pull request merged**